critiquebrainz/frontend/templates/review/revisions.html

```diff
@@ -42,6 +42,28 @@
 {% block scripts %}
   {{ super() }}
   <script>
+    {% if count > limit %}
+      var current_page = 0;
+      function load_more() {
+        var more_button = $("#more-button");
+        var loading_message = $("#loading-message");
+        more_button.hide();
+        loading_message.show();
+        $.ajax({
+              url: "{{ url_for('review.revisions_more', id=review.id) }}",
+              data: {page: ++current_page}
+            })
+            .done(function (data) {
+              loading_message.hide();
+              $("#results").append(data.results);
+              if (data.more === true) more_button.show();
+            })
+            .fail(function () {
+              alert({{ _('Failed to load older revisions!') | tojson }});
+            });
+      }
+    {% endif %}
```

Comment:
Can you explain why this function has to be outside the `$(document).ready(...)`?